plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -347,27 +349,29 @@ public void removeInstanceSnapshot(InstanceKey instanceKey) throws SnapshotExcep
         String registry = dockerInstanceKey.getRegistry();
         String repository = dockerInstanceKey.getRepository();
         if (registry == null || repository == null) {
+            LOG.error("Failed to remove instance snapshot: invalid instance key: " + instanceKey.getFields().toString());
```

Comment:
I would say it is preferable to log
LOG.error("Failed to remove instance snapshot: invalid instance key: {}", instanceKey);
instead of 
LOG.error("Failed to remove instance snapshot: invalid instance key: " + instanceKey.getFields().toString());